API.md

```diff
@@ -3200,8 +3200,7 @@ Implements functions to deal with the AsyncAPI document.
 <a name="module_@asyncapi/parser+AsyncAPIDocument+traverseSchemas"></a>
 
 #### asyncAPIDocument.traverseSchemas(callback, schemaTypesToIterate)
-Traverse schemas in the document and select which types of schemas to include.
-By default all schemas are iterated
+Traverse schemas in the document and select which types of schemas to include.
By default all schemas are iterated
```

Comment:
Where does this change come from? 🤔

Reply:
Don't know man. searching for it. I think it's some kind of newline conversion thing.